src/libs/Middleware/SaveResponseInOnyx.js

```diff
@@ -13,23 +15,30 @@ function SaveResponseInOnyx(response, request) {
                 return;
             }
 
+            // For most requests we can immediately update Onyx. For write requests we queue the updates and apply them after the sequential queue has flushed to prevent a replay effect in
+            // the UI. See https://github.com/Expensify/App/issues/12775 for more info.
+            let updateHandler = Onyx.update;
+            if (request.data.apiRequestType === CONST.API_REQUEST_TYPE.WRITE) {
+                updateHandler = QueuedOnyxUpdates.queueOnyxUpdates;
+            }
```

Comment:
Let's just use a ternary here, more straightforward

```suggestion
            const updateHandler = request.data.apiRequestType === CONST.API_REQUEST_TYPE.WRITE ? QueuedOnyxUpdates.queueOnyxUpdates : Onyx.update;
```

Reply:
Oh yeah haha, I forget about ternary expressions sometimes.